kibana/README.md

```diff
@@ -996,3 +996,64 @@ readonlyrest:
 ```
 
 Now try to login as user1, and ReadonlyREST Enterprise should initialise the index ".kibana\_user1" with all the index patterns and dashboards contained in the template tenancy.
+
+## Impersonation
+
+User with specific roles (`admin` or `unrestricted`) and permissions (`impersonation` section in the `readonlyrest.yml` file)
+can impersonate (imitate behaviors or actions of the user) another user.
+
+### Test settings
+
+During user impersonation, the `reasonlyrest.yml` settings are replaced by test settings and used during a whole impersonation process
+You need to configure test settings to be able to impersonate the user
+
+
+#### Setting test settings
+
+To set test settings:
+
+1. Open the RoR menu
+2. Click the Edit security settings button
+
+![Test settings ror menu](<../.gitbook/assets/test_settings_ror_menu.png>)
+
+3. Go into the Test settings tab
+4. You can set the TTL value. After the TTL value is reached, your test settings will be deactivated automatically
+5. You can load current settings as test settings
+6. You can deactivate settings manually
+7. You can save test settings as settings
+
+![test settings tab](<../.gitbook/assets/test_settings_tab.png>)
+
+### Impersonate
+It contains a list of external services and local users. You can add and edit the list of users in external services and impersonate a specific user.
+
+#### Local and auth mock services users configuration
+
+1. Open the RoR menu
+2. Click the Edit security settings button
+
+![Impersonate ror menu](<../.gitbook/assets/test_settings_ror_menu.png>)
+
+3. Go into the Impersonate tab
+4. You can free type impersonate. This button is available only in the situation when in some cases, the system is not able to receive all usernames. In this case, to impersonate, you need to type impersonating username manually.
```

Comment:
"You can freely type a username to impersonate using the dedicated button. This button will be available only when the system cannot anticipate all the possible usernames (for example, when ROR is configure to infer usernames from headers, etc)." 

Correct this if wrong, but the idea is to expand the explanation providing some scenarios.

Reply:
To me, looks good, one question to @coutoPL, is it the correct example :) ? `for example, when ROR is configure to infer usernames from headers, etc`

Reply:
You, as an admin, can use variables in ROR rules. There is a variable that allows you to extract value from headers. It can be used in context of some auth rule. In this case we cannot return users from the rule because their names are dynamic. 

So, yes - this is a correct example

Reply:
text changed ✔️